### PR TITLE
movie: Escape from Mars (1999) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Lame. Rating 1/5.

### CODE REVIEW

**Comment by odinlake** [comment]
Comment inline...

**Comment by paul-sheridan** [comment]
Come again?

**Comment by odinlake** [comment]
I write that when I left a comment attached to one of the lines in the
diff. You didn’t merge or reply to this one yet, so I think you missed my
comment - something about corporations and culture

**Comment by paul-sheridan** [comment]
Hum... I can't seem to find a record of the said attached comment anywhere. I sometimes notice comments when I click on the "Files Changed" tab, but I don't see anything there this time.

**Comment by odinlake** [comment]
This is what I see:

**Comment by odinlake** [comment]
This is still open...

File: notes/stories/film/film-scifi-1990s.st.txt [hunk 1]
```diff
@@ -11501,6 +11501,47 @@ biological weapons [The Drakh contaminated Earth with biogenic plague Earth usin
...
+https://en.wikipedia.org/wiki/Escape_from_Mars
+
+:: Ratings
+1 <paul> [TV film]
+
+:: Major Themes
+interplanetary space travel [The film follows five enterprising astronauts as they make the first manned trip to Mars.]
+colonization of Mars [Five astronauts went on a pioneering mission to Mars in order to lay the foundation for a human colony.]
+working together under stress [The five astronauts had to keep it together under various nail biting situations (e.g. a computer malfunction, a meteor shower, a breakdown of communications with Earth) in order to complete their mission of flying to Mars and back.]
+sociocultural issue [A point was hammered home about the first manned mission to Mars being overly commercialized.]
```
[the remark this answers is not in the document]
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1990s.st.txt:11527]: Visible at long last. After some thought, parenting aside, I think the **mass consumerism** theme well-enough covers what was going on here; will need to tweak the motivation (e.g. there was a corporate logo on the spaceship and numerous other such things I'd need to skim through the film again to check).

**Review by odinlake — APPROVED** [review]
Didn't I review this some time ago? I seem to remember themes and comments.

**Review by odinlake — APPROVED** [review]
Didn't I review this some time ago? I seem to remember themes and comments.

**Comment by paul-sheridan** [comment]
Ah, I replaced

sociocultural issue [A point was hammered home about the first manned mission to Mars being overly commercialized.]

with

mass consumerism [A point was hammered home about the first manned mission to Mars being overly commercialized. The spaceship was emblazoned with a corporate logo, Mars mission memorabilia was being peddled in news reports, etc.]

and left it at that; see my inline response.

did you still want to introduce the new theme?

**Comment by odinlake** [comment]
I realized that just as I submitted. The new theme is on the AHP branch 
- not a bad theme but either way. We don't like to leave themes without 
usage around.

**Pull request merged**